tests/controller/accountscontrollertest.php

```diff
@@ -203,17 +204,17 @@ public function testCreateAutoDetectFailure() {
 
 	public function newMessageDataProvider() {
 		return [
-			[false, false],
-			[true, false],
-			[false, true],
-			[true, true],
+			[false, false, false],
+			[true, false, false],
+			[false, true, false],
+			[true, true, true],
```

Comment:
Not really, but the tested method is so complex that I didn't want to write another test :see_no_evil: We really need to refactor it at some point.